### PR TITLE
Chapter 17-5 17_5_Usando_Scripts_en_Libwally.md

### PR DESCRIPTION
**Pull request by javiervargas**

### CODE REVIEW

File: es/17_5_Usando_Scripts_en_Libwally.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  ]
+}
+```
+El `vin` debe coincidir con la entrada que sustituyó en, pero es el `vout` lo que es emocionante: que ha creado una transacción con un `scripthash`!
+
+## Resumen: Usando scripts en Libwally
+
+Crear transacciones en Libwally es otro tema que podría ocupar todo un capítulo, pero lo bueno es que una vez que da este salto, puede introducir un script P2SH `PubKey`, y esa parte por sí sola es bastante fácil. Aunque la metodología detallada en este capítulo requiere que tenga una cadena hexadecimal de la transacción ya en la mano (probablemente creado con `bitcoin-cli`) si profundiza en Libwally, puede hacerlo todo usted mismo.
+
+> :fire: ***Cuál es el poder de los guiones en Libwally?*** Simplemente, puedes hacer algo que no podías hacer antes: crear una transacción bloqueada con un P2SH arbitrario.
```
Comment (icculp) [inline es/17_5_Usando_Scripts_en_Libwally.md:173]: que no podía

File: es/17_5_Usando_Scripts_en_Libwally.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  char *tx_hex;
+  lw_response = wally_tx_to_hex(tx,0, &tx_hex);
+
+  printf("%s\n",tx_hex);
+```
+Vamos a mostrar cómo hacer uso de eso en §17.7.
+
+## Pruebe su script de reemplazo
+
+Puede tomar el código de prueba del [directorio src](src/16_5_replacewithscript.c) y compilarlo:
```
Comment (icculp) [inline es/17_5_Usando_Scripts_en_Libwally.md:113]: ../src/16_5_replacewithscript.c

**Review by icculp — CHANGES_REQUESTED** [review]

**Review by icculp — APPROVED** [review]

**Comment by icculp** [comment]
@shannona ready to be merged

**Comment by shannona** [comment]
thank you

**Pull request merged**